src/applications/mhv/medications/tests/e2e/pages/MedicationsDetailsPage.js

```diff
@@ -337,22 +337,21 @@ class MedicationsDetailsPage {
     cy.get('[data-testid="no-image"]').should('contain', 'No image available');
   };
 
-  verifyCmopNdcNumberIsNull = prescriptionDetails => {
-    cy.intercept(
-      'GET',
-      `/my_health/v1/prescriptions/${
-        prescriptionDetails.data.attributes.prescriptionId
-      }`,
-      prescriptionDetails,
-    ).as('prescriptionDetails');
-    cy.get('@prescriptionDetails')
-      .its('response')
-      .then(res => {
-        expect(res.body.data.attributes).to.include({
-          cmopNdcNumber: null,
-        });
-      });
-  };
+  // verifyCmopNdcNumberIsNull = prescriptionDetails => {
+  //   cy.intercept(
+  //     'GET',
+  //     `/my_health/v1/prescriptions/${prescriptionDetails.data.attributes.prescriptionId
+  //     }`,
+  //     prescriptionDetails,
+  //   ).as('prescriptionDetails');
+  //   cy.get('@prescriptionDetails')
+  //     .its('response')
+  //     .then(res => {
+  //       expect(res.body.data.attributes).to.include({
+  //         cmopNdcNumber: null,
+  //       });
+  //     });
+  // };
```

Comment:
Will this block actually be needed later or can it be removed?

Reply:
I can remove it if this is an issue.